File: mycodes/1)arrays/my/p1/p1.py

def pairindex(arr,sum_1):
    for x in range(len(arr)):
        tmp = arr[x]

        for y in range(len(arr)):
            
            if arr[x] + arr[y] == sum_1:
                print("pair found at index {} and {} ({} + {})".format(x,y,arr[x] ,arr[y]))
                return







arr = [8,7,2,5,3,1]
sum_1 = 10
pairindex(arr,sum_1)

File: mycodes/1)arrays/p1/p1.c
#include <stdio.h>


int main()
{ 

  int sum = 10;

  int arr[] = {8,7,2,5,3,1};
  
  int size = sizeof(arr)/sizeof(int);


  for (int i = 0; i < size; ++i)
  {
   

		   for (int j = i+1; j < size; j++)
		   {
		  
		     
		     if (arr[i]+arr[j] == sum)
		     {
		     printf("pair found at index %d and %d (%d + %d)",i,j,arr[i] ,arr[j]);
		     return 1;
		     }
       

		   }


  }
   



	return 0;
}
